AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java

```diff
@@ -2206,7 +2206,7 @@ protected void displayCardAnswer() {
         // If the user wants to show the next question automatically
         if (mUseTimer) {
             long delay = mWaitQuestionSecond * 1000 + mUseTimerDynamicMS;
-            if (delay > 0) {
+            if (mWaitQuestionSecond > 0 && delay > 0) {
```

Comment:
As described above - I think this is simplest + correct

```suggestion
            if (mWaitQuestionSecond > 0) {
```